app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -111,4 +112,25 @@ def create_network_router_queue(userid, options = {})
     }
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
+
+  def create_security_group(options)
+    SecurityGroup.raw_create_security_group(self, options)
+  end
+
+  def create_security_group_queue(userid, options = {})
```

Comment:
@ladas, yes I thought about going DRY here too, meanwhile I believe this is out of scope for this PR. Could I suggest another pass to tackle this across the other wait_for_queue that have made it so far?